config/settings.yml

```diff
@@ -1376,6 +1376,7 @@ vanotify:
         evidence_submission_failure_email: fake_template_id
     ivc_champva:
       api_key: fake_secret
+      failure_email_threshold_days: 7
```

Comment:
This setting var is kinda pointless. It's only going to be used in one spot and if it needs updating either way you'll need a PR. I would only use the setting if you plan on adding it to the AWS ParamStore so that you can update it without a code change.

So either...
1. Create a slot in the ParamStore for it to avoid code changes if it needs updating
2. Just put the hardcoded value in the Sidekiq job

Reply:
Got it! Plan was to use AWS param, so should be good on that